node/Dockerfile

```diff
@@ -33,6 +33,10 @@ FROM clearlinux/os-core:latest
 
 COPY --from=builder /install_root /
 
+ENV NAME="node" \
+    VERSION="v12.13.0" \
+    PATH="/usr/bin:$PATH"
```

Comment:
The clear already had the env PATH defined. What is the value here to add one new PATH?

Reply:
will skip this info as it's already provided.